### ISSUE TITLE
Calls read chapters as missing chapters

### ISSUE THREAD

**Issue by Darkvramp** [tachiyomiorg/tachiyomi#6074]
### Steps to reproduce

If you have a large gap in read chapters say for example, you skip chapter 1 continue into chapter 2 and read all the way to chapter 16 marking them all read all the way there. Then backing out, expecting when you are going to add it to your library, then hit the continue reading button to be put back were you were(but I was not). So since there were 2 chapter ones in this manga, I used the slide scroll at the bottom of the page rather than actually scrolling through the manga. And then since I had already read all of that chapters it was going to skip from that chapter one to chapter 16, but the super odd thing was is, it marked all of those read chapters as missing. Some super fishy stuff that I know to be wrong or I'm reading the wrong side of the misty either. 

### Expected behavior

It should say chapters are already read. Not missing.

### Actual behavior

Chapters literally come up with a triangle log and the number of missing chapters.

### Tachiyomi version

tachiyomi j2k 1.3.2

### Android version

android 11

### Device

galaxy s10

### Other details

I was told by the j2k dev to put this here.
![Screenshot_20211009-082347_TachiyomiJ2K](https://user-images.githubusercontent.com/48525649/136665473-d948741e-4dcf-4785-b421-51566fb18bfc.jpg)
![Screenshot_20211009-082404_TachiyomiJ2K](https://user-images.githubusercontent.com/48525649/136665479-def9fddf-aade-42f8-8aac-0b430d7e106b.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
This is probably happening because you're reading with filtered chapters being skipped, so the reader thinks they're missing.

We could just reword this to "%d chapter gap".

**Comment by Darkvramp** [comment]
Yes I do.

**Comment by ghostbear** [comment]
It is just a warning to give you a heads up something is missing, it doesn't stop you from reading. And like @arkon said this sounds like you are using chapter filters or any other reader filter which triggers this

> We could just reword this to "%d chapter gap".

I would say that sounds way worse to me

**Comment by Riztard** [comment]
maybe "%d chapter missing or skipped"? 🤔

**Comment by Darkvramp** [comment]
Well the filters are doing their jobs, buuut....... they are not supposed to mark them as missing chapters. That is reserved for when there is a lack of chapters from the extension(source) in the first place. Not when iread the chapters and mark them as read. It's a bug that I feel your trying to pass off as nothing because it doesn't stop app use, for now.

**Comment by arkon** [comment]
It's working as intended — it notified you that you're potentially skipping chapters when reading. I'd consider rewording it, but I wouldn't remove it.

**Comment by Darkvramp** [comment]
> 
> 
> It's working as intended — it notified you that you're potentially skipping chapters when reading. I'd consider rewording it, but I wouldn't remove it.

but the warning for missing chapters is supposed to come on when the chapters are MISSING, not skipped.
